### ISSUE TITLE
[Radar] How do I set at least a triangle?

### ISSUE THREAD

**Issue by Brucia323** [recharts/recharts#4727]
## Do you want to request a feature or report a bug?

Request a feature.

## What is the current behavior?

I'm working on a tool that records the user's diet and that will present a radar chart. If the user only records one meal, the radar chart becomes a line, and I use a bit of code to increase the length of the data so that it is at least a triangle.

```tsx
  while (mealEnergiesChart.length < 3) {
    mealEnergiesChart.push({
      label: '',
      value: 0,
      fullMark: 100,
    });
  }
```

I do get a triangle, but the radar chart doesn't show up. After trying, it should be caused by the same label. There is an 'allowDuplicatedCategory' property in the docs, the default value is true, and I didn't change it.

Now I use the following code and it looks fine, but I want to ask, is there a better way to do it? Or provide an API?

```tsx
  while (mealEnergiesChart.length < 3) {
    mealEnergiesChart.push({
      label: new Array(mealEnergiesChart.length).fill(' ').join(''),
      value: 0,
      fullMark: 100,
    });
  }
```

**Comment by ckifer** [comment]
Hi! I'm a bit confused. You are trying to ensure that there is always a triangle even when there is only one datapoint? Are the other two datapoints that you add meaningless then? 

Can you reproduce your situation in codesandbox?

**Comment by Brucia323** [comment]
Yes, triangles can only be achieved now by adding meaningless data points.

[![Edit recharts radar](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/recharts-radar-wz7nqq)

You can see what it looks like when it is not filled into triangles by removing the while loop. If you set the label provided in the while loop to an empty string, you can also see that `allowDuplicatedCategory` may not work.

**Comment by ckifer** [comment]
Thanks the example helps. 

So you essentially want a way to provide a minimum number of points even if those points are empty.
